docs/api/platform/Error.md

```diff
@@ -298,12 +329,99 @@ void save_all_errors() {
     mbed_clear_all_errors();
 }
 ```
+
+#### Using `mbed_get_reboot_error_info()` to retrieve the reboot error info
+The error context captured by the error handling system can be retrieved using mbed_get_reboot_error_info() API. See the below code for example usage of that API. 
+In the example below, a status variable reboot_error_detected has been used to track the presence of error context capture.
+
+```CPP TODO
```

Comment:
What are we missing for the snipping compilation script to run this so we can remove the TODO?